src/signHash.c

```diff
@@ -40,16 +40,25 @@ static void do_approve(void)
 		// Send the data in the APDU buffer, which is a 64 byte signature.
 		assert(IO_APDU_BUFFER_SIZE >= SCHNORR_SIG_LEN_RS);
 		io_exchange_with_code(SW_OK, SCHNORR_SIG_LEN_RS);
+#ifdef HAVE_BAGL
 		// Return to the main screen.
 		ui_idle();
+#else
+		nbgl_useCaseStatus("TRANSACTION\nSIGNED", true, ui_idle);
+#endif
 }
 
 static void do_reject(void)
 {
     io_exchange_with_code(SW_USER_REJECTED, 0);
+#ifdef HAVE_BAGL
     ui_idle();
+#else
+    nbgl_useCaseStatus("Transaction rejected", false, ui_idle);
+#endif
```

Comment:
Should the blocks here not be interchanged:
```suggestion
#ifdef HAVE_BAGL
    nbgl_useCaseStatus("Transaction rejected", false, ui_idle);
#else
    ui_idle();
#endif
```
or am I missing something?

Reply:
Nevermind, reading the makefile I see that I got the wrong end of the stick and that `BAGL` is for Nano S etc. not Stax.